lib/swoosh/adapters/sendgrid.ex

```diff
@@ -103,7 +103,7 @@ defmodule Swoosh.Adapters.Sendgrid do
     Map.put(body, :content, content)
   end
   defp prepare_content(body, %{html_body: html}), do: Map.put(body, :content, [%{type: "text/html", value: html}])
-  defp prepare_content(body, %{text_body: text}), do: Map.put(body, :content, [%{type: "text/plain", type: text}])
+  defp prepare_content(body, %{text_body: text}), do: Map.put(body, :content, [%{type: "text/plain", value: text}])
```

Comment:
Actually I think the code you have there already works. I just need to remove these two additional functions